### PR TITLE
Switch timing and pid in tracelib using platform/ mechanism

### PR DESCRIPTION
**Pull request by piotrswigon**
This is to improve the compilation for the browser internally.

### CODE REVIEW

File: src/platform/time-node.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+// High resolution time in micros elapsed since the beginnig of time, which is arbitrary
```
Comment (mmandlis) [inline src/platform/time-node.ts]: typo - beginning

File: src/platform/time-node.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+// but consistent throughout invocations of the Arcs Runtime instantiation.
+export function now() {
```
Comment (mmandlis) [inline src/platform/time-node.ts]: we already have platform based `now()` methods in date-node.ts and date-web.ts files, which might be a bit confusing. 
maybe worth consolidating them somehow? or at least maybe you could add a comment in the `date-` files?

File: src/platform/time-web.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+ * @license
+ * Copyright 2020 Google LLC.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+// High resolution time in micros elapsed since the beginnig of time, which is arbitrary
```
Comment (mmandlis) [inline src/platform/time-web.ts]: typo - beginning

File: src/platform/time-node.ts (hunk 2 above)
Comment (piotrswigon) [reply src/platform/time-node.ts]: Thanks for letting me know! I've removed the time-* variants!

File: src/platform/time-web.ts (hunk 3 above)
Comment (piotrswigon) [reply src/platform/time-web.ts]: fixed... and removed :)

File: src/tracelib/trace.ts [hunk 4]
```diff
@@ -9,6 +9,8 @@
  */
 
 import {Dictionary, Producer, Predicate} from '../runtime/hot.js';
+import {now as nowMillis} from '../platform/date-web.js';
+import {pid} from '../platform/process-web.js';
```
Comment (mmandlis) [inline src/tracelib/trace.ts]: is `pid` still used anywhere?

Comment (piotrswigon) [reply src/tracelib/trace.ts]: yes, it's used below. I believe it's required in the message format for the traces to be opened in the chrome tracing view.

**Pull request merged**